providers/implementations/include/prov/names.h

```diff
@@ -231,8 +231,12 @@
 #define PROV_NAMES_KECCAK_384 "KECCAK-384"
 #define PROV_NAMES_KECCAK_512 "KECCAK-512"
 
-#define PROV_NAMES_SHAKE_128 "SHAKE-128:SHAKE128:2.16.840.1.101.3.4.2.11"
-#define PROV_NAMES_SHAKE_256 "SHAKE-256:SHAKE256:2.16.840.1.101.3.4.2.12"
```

Comment:
Hmm... I am afraid this is an API break. I suspect there will be a breakage if you run this PR with an older FIPS provider and load both default and the old fips provider. There will be a naming mismatch or it will make the  PROV_NAMES_SHAKE_128_256 and PROV_NAMES_SHAKE_128 the same. Similarly for SHAKE-256 and SHAKE-256/512.

Reply:
It turns out that it is even worse, Moving the oid breaks even the running tests..
I assume in the name mapping that it maps the first name back to an oid? This sounds like a bad thing to be doing.
So when I try selecting SHAKE-128/256 it go tthe SHAKE-128 table entry.

So I have taken the OID change out of this PR. BUT this is eventually going to bite us at some point.. If the OID in a signature was SHAKE-128 and we tried interoping with something else we are going to get the wrong thing potentially if it assumes that the size is supposed to be 256 bit (due to the fact that our implementation incorrectly choses a default of 128 bits currently). We may need to do a breaking change to fix this..